bin/ci/_includes.sh

```diff
@@ -78,6 +78,11 @@ export DRUPAL_TEST_WEBDRIVER_CHROME_ARGS="--disable-gpu --headless --no-sandbox"
 export DRUPAL_TEST_WEBDRIVER_HOSTNAME="localhost"
 export DRUPAL_TEST_WEBDRIVER_PORT="4444"
 
+# Set TMPDIR on GitHub Actions
+if [[ "$RUNNER_TEMP" ]]; then
```

Comment:
The Coveralls command in the after_success script relies on TMPDIR being set, but this was a TravisCI-ism. It's not set on GH Actions.

Reply:
Great. However, before this change, the variable in that script (https://github.com/acquia/orca/blob/v3.11.0/bin/ci/after_success.sh#L25) was overridable--via bash's (ridiculous, convoluted [parameter expansion](https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts/122878) 🙄 ). Adding a commit to restore that functionality.